tema2sapt2/tema2s2/Program.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Exercise 2 <<<<<<<<<<<<<<<<<<<<<<<<<>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
+Console.WriteLine("Exercise 2");
+
+int[,] arraynumbers = new int[4, 3] { { 123, 345 ,567}, { 12, 56 ,-12}, { 34, -98,65 }, { 765, 543,321 } };
```

Comment:
See previous comment about naming - camel case means that we would need to name this variable `arrayNumbers`  which makes it easier to read.